src/DiffEqGPU.jl

```diff
@@ -281,7 +292,28 @@ function batch_solve(ensembleprob,alg,ensemblealg::EnsembleArrayAlgorithm,I;kwar
 
     u0 = reduce(hcat,Array(probs[i].u0) for i in 1:length(I))
     p  = reduce(hcat,probs[i].p isa SciMLBase.NullParameters ? probs[i].p : Array(probs[i].p)  for i in 1:length(I))
-    sol, solus = batch_solve_up(ensembleprob,probs,alg,ensemblealg,I,u0,p;kwargs...)
+
+    sol, solus = 
+    if ensemblealg isa EnsembleGPUArrayAutonomous
+        ps = CuArray([SVector{length(probs[i].p)}(probs[i].p) for i in 1:length(I)])
+        if typeof(alg) <: OrdinaryDiffEq.Tsit5
```

Comment:
Could this be checked without importing `OrdinaryDiffEq`?

Reply:
We should just make a different algorithm here.